systemtest/src/test/java/io/strimzi/systemtest/NamespaceIT.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.strimzi.test.JUnitGroup;
+import io.strimzi.test.Namespace;
+import io.strimzi.test.OpenShiftOnly;
+import io.strimzi.test.StrimziRunner;
+import io.strimzi.test.k8s.KubeClient;
+import io.strimzi.test.k8s.Oc;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+import static io.strimzi.systemtest.NamespaceIT.NAMESPACE1;
+import static io.strimzi.systemtest.NamespaceIT.NAMESPACE2;
+import static io.strimzi.test.TestUtils.map;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(StrimziRunner.class)
+@Namespace(NAMESPACE1)
+@Namespace(NAMESPACE2)
+public class NamespaceIT extends AbstractClusterIT {
+
+    private static final Logger LOGGER = LogManager.getLogger(NamespaceIT.class);
+
+    public static final String NAMESPACE1 = "kafka-cluster-test";
+    public static final String NAMESPACE2 = "kafka-cluster-test2";
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @OpenShiftOnly
+    public void testForWorkWithMultipleNamespaces() {
+        String clusterName = "openshift-my-cluster";
+
+        //Deploying cluster in first namespace
+        KubeClient client1 = kubeClient.inNamespace(NAMESPACE1);
+        client1.deployCOFromCommandLine(NAMESPACE1, "../examples/install/cluster-operator");
+        client1.waitForDeployment("strimzi-cluster-operator");
+        client1.installTemplates(NAMESPACE1, "../examples/templates/cluster-operator");
+        Oc oc1 = (Oc) client1;
+        oc1.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));
+        client1.waitForStatefulSet(zookeeperClusterName(clusterName), 3);
+        client1.waitForStatefulSet(kafkaClusterName(clusterName), 3);
+
+        //Deploying cluster in second namespace
+        KubeClient client2 = kubeClient.inNamespace(NAMESPACE2);
+        client2.deployCOFromCommandLine(NAMESPACE2, "../examples/install/cluster-operator");
```

Comment:
Working with multi-namespaces means that the CO can live in just one namespace and then watching cluster ConfigMap created in different namespaces and deploying the clusters there. In this test you are deploying the CO twice in both the namespaces where you want to deploy the clusters. Is it something that you really wanted ?